### PR TITLE
Automatic sortable table class

### PR DESCRIPTION
**Pull request by levithomason**

### CODE REVIEW

File: test/specs/collections/Table/Table-test.js [hunk 1]
```diff
@@ -106,4 +106,22 @@ describe('Table', () => {
         });
     });
   });
+
+  describe('sortable', () => {
```
Comment (jhabdas) [inline test/specs/collections/Table/Table-test.js]: Haven't seen one of these in a while.

File: src/collections/Table/Table.js [hunk 2]
```diff
@@ -82,6 +82,7 @@ export default class Table extends Component {
     const classes = classNames(
       'sd-table',
       'ui',
+      {sortable: !!this.props.onSortChange},
```
Comment (levithomason) [inline src/collections/Table/Table.js:85]: If the sort handler is present, add the class so we get the proper formatting:

http://semantic-ui.com/collections/table.html#sortable

File: test/specs/collections/Table/Table-test.js (hunk 1 above)
Comment (jhabdas) [inline test/specs/collections/Table/Table-test.js]: ;)

File: test/specs/collections/Table/Table-test.js [hunk 3]
```diff
@@ -106,4 +106,22 @@ describe('Table', () => {
...
+
+    describe('without onSortChange prop', () => {
+      it('does not have class "sortable"', () => {
+        render(<Table data={tableData} />)
+          .findClass('sd-table')
+          .props.className
+          .should.not.include('sortable');
+      });
+    });
+  });
```
Comment (levithomason) [inline test/specs/collections/Table/Table-test.js:126]: Assert the class is added/removed along with the sort handler prop.

**Comment by ghost** [comment]
:hamburger:

**Pull request merged**